express/src/modules/CanonCamera/CanonCameraDevice.ts

```diff
@@ -0,0 +1,191 @@
+import { IHardware } from "../../interfaces/IHardware";
+import { Guid } from "guid-typescript";
+import { HardwareProperty } from "../../models/hardware-property";
+import { IStatus } from "../../interfaces/IStatus";
+import { ISubscription } from "../../interfaces/ISubscription";
+import { Logger } from "../../common/logger";
+import { CanonCameraAPI } from "./CanonCameraAPI";
+import { CanonCameraCaptureData } from "./models/canoncamera-capture-data";
+
+export class CanonCameraDevice implements IHardware {
+	/**
+	 * Public Member Variables
+	 */
+	public id: Guid;
+	public modelName: string;
+	public serial: string;
+	public type: string = "Canon Camera";
+
+	/**
+	 * Private Variables
+	 */
+	private _autoFocus: boolean = true;
+
+	get timeout(): number {
+		return 2 * 60 * 1000;
+	}
+	/**
+	 * Constructor
+	 */
+	constructor() {
+		this.id = Guid.create();
+	}
+
+	/**
+	 * Public Functions
+	 */
+	public GetProperties(): Promise<Array<HardwareProperty>> {
+		return new Promise<Array<HardwareProperty>>(async (resolve, reject) => {
+			const properties: Array<HardwareProperty> = Array<HardwareProperty>();
+
+			try {
+				const zoomProperty: HardwareProperty = new HardwareProperty();
+				zoomProperty.id = "zoom";
```

Comment:
put this is its own function like how the other modules work and the auto focus property works